site/themes/s2b_hugo_theme/static/css/custom.css

```diff
@@ -71,6 +71,22 @@ main .donate a {
   }
 }
 
+.customers .item {
+  margin-top: 2px;
+  margin-bottom: 2px;
+}
+
+.customers .item:focus-within {
+  outline: 2px solid #337AB7;
```

Comment:
Would be cool to move colors to CSS variables.

Reply:
Added #480.